core/src/test/java/com/opensymphony/xwork2/DefaultUnknownHandlerManagerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.opensymphony.xwork2;
+
+import com.opensymphony.xwork2.config.entities.ActionConfig;
+import com.opensymphony.xwork2.config.providers.SomeUnknownHandler;
+
+import java.util.ArrayList;
+
+/**
+ * Partial test to the DefaultUnknownHandlerManager to understand the relationship between Manager and Handlers.
+ *
+ * @author Ziyad Alsaeed
```

Comment:
Apache doesn't accept author tag anymore. Instead see [here](https://www.apache.org/foundation/how-it-works.html#roles) if you're interested to contribute :)